test/sets.jl

```diff
@@ -363,6 +363,19 @@ function runtests()
     end
 end
 
+function test_interval_promote()
+    for S in [MOI.Interval, MOI.Semiinteger, MOI.Semicontinuous]
```

Comment:
```suggestion
    for S in (MOI.Interval, MOI.Semiinteger, MOI.Semicontinuous)
```
It's okay in this case, but I always try to avoid `[]` for these sorts of heterogeneous collections because it promotes to a common argument. `[1, x, 2x + 3]` is a great example because they all get promoted to `AffExpr` so you don't actually test different values.